includes/admin/class-admin-settings.php

```diff
@@ -91,6 +91,16 @@ public static function save() {
 				die();
 			}
 
+			// Show error message if Akismet not configured and Admin try to save 'enabled' option.
+			if ( isset( $_POST['akismet_spam_protection'] )
+			     && 'enabled' === $_POST['akismet_spam_protection']
```

Comment:
@jaydeeprami Can you use` give_is_setting_enabled`?

Reply:
Sure @ravinderk

Reply:
@ravinderk, I have updated it.